tests/APIv2/UsersTest.php

```diff
@@ -129,6 +129,37 @@ public function testDeletePhoto() {
         $this->assertStringEndsWith('/applications/dashboard/design/images/defaulticon.png', $user['photoUrl']);
     }
 
+    /**
+     * Test confirm email is successful
+     */
+    public function testConfirmEmailSucceed() {
+        $emailKey = ['emailKey' => 'test123'];
+        $user = $this->testPost();
+
+        /** @var \UserModel $userModel */
+        $userModel = self::container()->get('UserModel');
+        $userModel->saveAttribute($user['userID'], 'EmailKey', 'test123');
+
+        $response = $this->api()->post("{$this->baseUrl}/{$user['userID']}/confirmEmail", $emailKey);
+        $this->assertEquals($response->getBody(), ['emailConfirmed' => true]);
```

Comment:
I believe users created with the users API endpoint are going to have their email address confirmed by default. In that case, this test would confirm the request didn't trigger an exception, but not that it necessarily affected the value of `emailConfirmed`.